enricher/api/src/main/java/io/fabric8/maven/enricher/api/AbstractHealthCheckEnricher.java

```diff
@@ -30,25 +32,67 @@ public AbstractHealthCheckEnricher(EnricherContext buildContext, String name) {
         super(buildContext, name);
     }
 
+    // Available configuration keys
+    protected enum Config implements Configs.Key {
+
+        probeMode {{ d = "all";}};
```

Comment:
We can decide to enrich all containers here, or we can rely here on the concept of "project's main container", or list the containers that should be enriched using a configuration option (also a global one if this enricher is not the only one that should be scoped to a specific container). I'd not go for "first" / "last" as it relies too much on what the current implementation does but it may change in the future (with plugin updates or with "fragments" added in other ways e.g. #941).